pkg/hubble/parser/cell/cell.go

```diff
@@ -71,7 +72,7 @@ func newPayloadParser(params payloadParserParams) (parser.Decoder, error) {
 			params.Config.SkipUnknownCGroupIDs,
 		),
 	)
-	return parser.New(params.Log, g, g, g, params.Ipcache, g, params.LinkCache, params.CGroupManager, parserOpts...)
+	return parser.New(params.Log, g, g, g, params.Ipcache, g, params.LinkCache, params.CGroupManager, params.PolicyRepo, parserOpts...)
```

Comment:
Maybe we could export the bakery directly from the policy cell? I'm trying to reduce dependency on the repo.